src/seedu/addressbook/AddressBook.java

```diff
@@ -1,12 +1,4 @@
 package seedu.addressbook;
-
```

Comment:
May I know why you removed all the header comments from the file? Please check the module handbook regarding the need for header and class comments.